server/@types/launchpad/index.d.ts

```diff
@@ -70,11 +70,10 @@ export type UpdatedTokensResponse = {
 
 export type Establishment = {
```

Comment:
This type was not actually used anywhere although it looks like this was it's original purpose.